src/services/doppler-api-client.test.ts

```diff
@@ -382,4 +382,57 @@ describe('HttpDopplerApiClient', () => {
       expect(result.value.items[0].email).toEqual('test@fromdoppler.com');
     });
   });
+
+  describe('GetCampaignsSummaryResults', () => {
+    it('should get and error', async () => {
```

Comment:
This case also covers for incorrect id for campaign right?

Reply:
yes!